packages/@orbit/record-cache/src/live-query/utils.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Record,
+  cloneRecordIdentity,
+  RecordIdentity,
+  RecordOperation
+} from '@orbit/data';
+
+export interface RecordChange extends RecordIdentity {
+  keys: string[];
+  attributes: string[];
+  relationships: string[];
+  remove: boolean;
+}
+
+export function recordOperationChange(
```

Comment:
@tchak this impl. is 🔥🔥🔥 Was able to get an optimal implementation of react-orbit hook with this record change detection on the source's `transform` event instead of cache's `patch` event.